Tasks/Lavyshyk_Olga/homework1/task1.py

```diff
@@ -0,0 +1,13 @@
+sum1 = int(input("Enter deposit amount = "))
+y = int(input("Enter number of years = "))
+N = float(input("Annual interest = "))
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point